3648/1/gem_stat.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'open-uri'
+require 'httparty'
+
+AUTH_TOKEN = YAML.load_file('tokens.yml')['AUTH_TOKEN']
+
+class GemStatistic
+  attr_reader :gem_name
+  def initialize(gem_name)
+    @gem_name = gem_name
+  end
+
+  def create_stat
+    {
+      gem_name: gem_name,
+      used_by: find_used_by,
+      watch_by: find_wathed_by,
```

Comment:
Be attentive to details. You made another spelling mistake and didn't rename field to past tense

Reply:
This spelling mistake trailing further in code